AppConnectSwift/CaptureImageViewController.swift

```diff
@@ -82,6 +82,13 @@ class CaptureImageViewController: UIViewController, UIImagePickerControllerDeleg
                                     datastore = nil
                                 }
                             }
+                            else {
+                                NSOperationQueue.mainQueue().addOperationWithBlock {
+                                    self.showAlert("Unable to save data", message: "Unable to save data as \(err)")
```

Comment:
Since we already have "Unable to save data" in the title, maybe the message could just be the error?